packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -84,7 +84,7 @@ import { computed, defineComponent, InputHTMLAttributes, PropType, ref, toRefs }
 import { useFormProps } from '../../composables/useForm'
 import { useValidation, useValidationProps, useValidationEmits, ValidationProps } from '../../composables/useValidation'
 import { useCleave, useCleaveProps } from './hooks/useCleave'
-import { useFocus } from '../../composables/useFocus'
+import { useFocusDeep } from '../../composables/useFocusDeep'
```

Comment:
To prevent empty div we should update line 30 like this:
`<template v-if="success || computedError || canBeCleared || loading" #icon="slotScope">`